docs/source/quick_start.rst

```diff
@@ -6,21 +6,41 @@ the box there is support for pulling data from a multitude of
 `data sources <data_sources.html>`_ including Beiwe, XNAT, REDCap, 
 Dropbox, Box, Mediaflux, Daris, RPMS, external hard drives, and more.
 
+
+
 Installation
 ------------
 Just use ``pip`` ::
 
     pip install lochness
 
 
-or for most recent DPACC-lochness ::
+For most recent DPACC-lochness ::
 
     pip install git+https://github.com/PREDICT-DPACC/lochness
 
 
+For debugging ::
+
+    cd ~
+    git clone https://github.com/PREDICT-DPACC/lochness
+    pip install -r ~/lochness/requirements.txt
+
+    export PATH=${PATH}:~/lochness/scripts  # add to ~/.bashrc
+    export PYTHONPATH=${PYTHONPATH}:~/lochness  # add to ~/.bashrc
```

Comment:
Same comment as above

Reply:
Thanks for this. Removed from the `quick_start.rst`